apps/vr-tests/package.json

```diff
@@ -36,9 +36,9 @@
     "@fluentui/react-theme": "^9.0.0-alpha.15",
     "@fluentui/scripts": "^1.0.0",
     "@fluentui/storybook": "^1.0.0",
-    "@storybook/addons": "6.0.28",
-    "@storybook/channels": "6.0.28",
-    "@storybook/react": "6.0.28",
+    "@storybook/addons": "6.2.9",
+    "@storybook/channels": "6.2.9",
+    "@storybook/react": "6.2.9",
```

Comment:
Remove these since they're now declared at the root
```suggestion
```

Reply:
Removing these seems to have caused a build failure 🤔

Reply:
What's the error?

Reply:
![image](https://user-images.githubusercontent.com/8649804/124316852-16f28000-db2b-11eb-939f-f19e3a9354e9.png)